docs/tutorial/Pre-requisites of running a Waku node.md

```diff
@@ -0,0 +1,106 @@
+If you already have an Ethereum account with sufficent ethers on Goerli testnet then you can skip the first two sections.
+## Creating Goerli Ethereum Account and obtaining its private key
+
+
+1. Download and install Metamask [https://metamask.io/download/](https://metamask.io/download/)
+2. Create a new wallet and save your secret recovery key
+    
+    ![](https://i.imgur.com/HEOI0kp.jpg)
+
+3. Login to Metamask
+    
+    ![](https://i.imgur.com/zFduIV8.jpg)
+
+4. You should already see an account created. As you can see on top right, it should be pointing to Ethereum mainnet.
+    
+    ![](https://i.imgur.com/gk3TWUd.jpg)
+
+5. You can use the same account for different networks. For Waku we need to connect to the Goerli test network.
+6. You can switch to a test network by clicking on the drop down menu. Select Goerli test network for Waku:
+
+    ![](https://i.imgur.com/kbuup6y.jpg)
+
+7. Click on Show/hide test networks
+8. Enable “Show Test Networks
+    
+    ![](https://i.imgur.com/02x7iSX.jpg)
+    
+    ![](https://i.imgur.com/UFsoRHR.jpg)
+
+9. Close the settings and now you should see the test networks in the drop down on the top right
```

Comment:
please change the picture so that the Goerli network is check-marked instead of the ethereum mainnet.

Reply:
Please add the period `.` to the end of all the lines (including here) for the consistency.